lib/login.js

```diff
@@ -33,7 +34,7 @@ module.exports = async function login() {
 
     return mgmtToken;
   } catch (err) {
-    console.log('Failed to login into Contentful');
+    console.log(chalk.red('Failed to login into Contentful.'));
```

Comment:
Is red going to look in dark editors?

Reply:
It's using the `red` of the theme, so it'll be readable. Chalk handles everything.